<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BOZ0030</class>
 <widget class="QDialog" name="BOZ0030">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BOZ0030</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_vstup">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>401</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Vloz text</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_sifra">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>401</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_zasifrovat">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Zasifrovat</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_desifrovat">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Desifrovat</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_desifrovat">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>401</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
